### ISSUE TITLE
About finetuning performance

### ISSUE THREAD

**Issue by CARRLIANSS** [Pang-Yatian/Point-MAE#39]
Hi Yatian, 

I really appreciate your great work. I have checked the pre-training logs [here](https://github.com/Pang-Yatian/Point-MAE/releases/download/main/logs.tar) you provided. I found that from epoch3 to epoch300 the loss only dropped from 9.1 to 2.6. I have the following two questions to ask you: 
(1) What are the results of fine-tuning based on epoch3? Will the performance drop a lot? 
(2) Why did not you consider using the test set to verify its generalization ability during the pre-training stage?

Thanks a lot!

**Comment by Pang-Yatian** [comment]
Hi, 

1) Generally, in self-supervised learning, the pre-training loss does not relate to the final finetuning performance. I am sorry that we did not do any experiments finetuning from epoch3 or others. I suppose the performance will drop a lot.

2) I assume you are referring to downstream task test sets. Remember we are doing pre-training in a self-supervised manner, which means no labels are provided and the model does not have the ability to do classification using the test set.
